postem/postem-app/pom.xml

```diff
@@ -7,124 +7,143 @@
     <version>22-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
+  
   <name>Sakai Postem Tool</name>
   <groupId>org.sakaiproject</groupId>
   <artifactId>sakai-postem-tool</artifactId>
   <packaging>war</packaging>
 
-    <!-- profiles added for WAS support -->
-    <profiles>
-        <profile>
-            <id>full</id>
-            <activation>
-                <activeByDefault>true</activeByDefault>
-            </activation>
-            <dependencies>
-                <dependency>
-                  <groupId>org.sakaiproject.jsf</groupId>
-                  <artifactId>myfaces-tool</artifactId>
-                  <type>pom</type>
-		  <exclusions>
-		    <!-- exclude this to avoid the collision btw RI and tomahawk -->
-		    <exclusion>
-		      <groupId>javax.faces</groupId>
-		      <artifactId>jsf-api</artifactId>
-		    </exclusion>
-		    <exclusion>
-		      <groupId>javax.faces</groupId>
-		      <artifactId>jsf-impl</artifactId>
-		    </exclusion>
-		  </exclusions>
-                </dependency>
-            </dependencies>
-        </profile>
-        <profile>
-            <!-- exclude xerces xml parser apis for WAS -->
-            <id>was</id>
-            <dependencies>
-                <dependency>
-                  <groupId>org.sakaiproject.jsf</groupId>
-                  <artifactId>myfaces-tool</artifactId>
-                  <type>pom</type>
-		  <exclusions>
-		    <!-- exclude this to avoid the collision btw RI and tomahawk -->
-		    <exclusion>
-		      <groupId>javax.faces</groupId>
-		      <artifactId>jsf-api</artifactId>
-		    </exclusion>
-		    <exclusion>
-		      <groupId>javax.faces</groupId>
-		      <artifactId>jsf-impl</artifactId>
-		    </exclusion>
-		  </exclusions>
-                </dependency>
-            </dependencies>
-        </profile>
-    </profiles>
+    <properties>
+        <maven.deploy.skip>true</maven.deploy.skip>
+        <skipNexusStagingDeployMojo>true</skipNexusStagingDeployMojo>
+        <thymeleaf.version>3.0.11.RELEASE</thymeleaf.version>
+    </properties>
+
+    <developers>
+        <developer>
+            <name>Miguel Pellicer</name>
+            <email>mpellicer@edf.global</email>
+        </developer>
+    </developers>
+
+    <dependencies>
```

Comment:
Review all of these dependencies and remove the unused ones.

Reply:
Thank you for your comment.
I have revised the dependencies and deleted the not used ones.